WordPress/Classes/ViewRelated/Stats/Subscribers/StatsSubscribersStore.swift

```diff
@@ -106,17 +107,20 @@ struct StatsSubscribersStore: StatsSubscribersStoreProtocol {
                 completion(.failure(error))
             } else {
                 // Combine both wpcom and email subscribers into a single list
-                let followers = Array(
-                    followers
-                        .sorted(by: { $0.subscribedDate > $1.subscribedDate })
-                        .prefix(quantity)
-                )
-                completion(.success(followers))
+                let combinedSubscribers = Array((wpComFollowers?.topDotComFollowers ?? []) + (emailFollowers?.topEmailFollowers ?? [])
```

Comment:
Based on the answer here pejTkB-1l5-p2#comment-1316 the idea is for all of this aggregation to be replaced by `v2/sites/{site_id}/stats/subscribers`. I think we can do that both in scope and outside the scope of the project depending on time constraints.